frontend/src/routes/console.tsx

```diff
@@ -13,6 +13,7 @@ import {
   CustomPages,
   CoursesManage
 } from 'console/components';
+import { LogosSideBar as newLogos } from 'newConsole/components';
```

Comment:
Instead of `newLogos`, use `NewLogoSidebar` instead, since there is a seperate `Logos` component.

Reply:
Done like so.